setup/createVirtualEnvPy26.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/zsh
```

Comment:
while this script may not be used presently I am not sure it should be removed.  Generally reducing functionality is unwise.